python-api-examples/offline-omnilingual-asr-ctc-decode-files.py

```
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)